reference/errors/error_codes.md

```diff
@@ -54,6 +54,10 @@ An index with this UID was not found, check out our guide on [index creation](/l
 
 The requested index already has a primary key that cannot be changed.
 
+## `index_primary_key_multiple_candidates_found`
+
+[Primary key inference](/learn/core_concepts/primary_key.md#meilisearch-guesses-your-primary-key) failed as the index contains multiple fields ending with `id`. Use the [update index endpoint](/reference/api/indexes.md#update-an-index) to manually set a primary key.
```

Comment:
```suggestion
[Primary key inference](/learn/core_concepts/primary_key.md#meilisearch-guesses-your-primary-key) failed because the received documents contain multiple fields ending with `id`. Use the [update index endpoint](/reference/api/indexes.md#update-an-index) to manually set a primary key.
```